lib/ansible/modules/network/nxos/nxos_ospf_vrf.py

```diff
@@ -90,6 +90,13 @@
     description:
       - Specifies the reference bandwidth used to assign OSPF cost.
         Valid values are an integer, in Mbps, or the keyword 'default'.
+  bfd:
+    description:
+      - Enables BFD on all OSPF interfaces.
+      - "Dependency: 'feature bfd'"
+    version_added: "2.9"
+    type: bool
```

Comment:
Would you please change this to type str which choices `[disable, enable]`. This would make bfd type consistent across all the modules.